README.md

```diff
@@ -33,6 +33,8 @@ public class Example {
 }
 ```
 
+The program prints:
+
 ```plaintext
```

Comment:
I think you can change these back to `text`. See my Slack message - I asked Liz about this and she recommended, for consistency, to use the `text` versus `plaintext`. I'm going through my pull requests now and changing them.

Reply:
ok, updated